chapters/location_component_interface.adoc

```diff
@@ -0,0 +1,146 @@
+// Copyright 205 The Khronos Group, Inc.
+// SPDX-License-Identifier: CC-BY-4.0
+
+ifndef::chapters[:chapters:]
+ifndef::images[:images: images/]
+
+[[location-and-component-interface]]
+= Location and Component Interface
+
+This chapter is an overview of the link:https://docs.vulkan.org/spec/latest/chapters/interfaces.html#interfaces-iointerfaces-locations[Location and Component Assignment] chapter to help give examples, especially around some of the more extreme edge cases.
+
+The simplest way to think about a `Location` is that it is made up of four 32-bit `Components`.
+This means a `vec4`/`float4`/`uvec4`/etc will fit perfectly in a single `Location`.
+Multiple variables can be packed into the same `Location` if they are specified for unique components.
```

Comment:
Maybe it's better to reuse the Vulkan spec language here as an aid when reading that chapter.
```suggestion
Multiple variables can be packed into the same `Location` if their `Component`s do not overlap.
```